rust/lib/Cargo.toml

```diff
@@ -1,15 +1,14 @@
 [package]
 name = "rustlib"
-version = "1.0.0"
-authors = ["Aditya Kulkarni <aditya@zecwallet.co>"]
+version = "1.0.1"
+authors = ["Zingolabs <zingo@zingolabs.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 jni = { version = "0.10.2", default-features = false }
-zecwalletlitelib = { git="https://github.com/zingolabs/zecwallet-light-cli", default-features=false, rev="5e73adef4615786954f97f80e6fbbcd650ee39c7" }
-#zecwalletlitelib = { path = "../../../zecwallet-light-cli/lib", default-features=false }
+zecwalletlitelib = { git="https://github.com/zingolabs/zecwalletlitelib", default-features=false, rev="3b2ef3a72df6f740bf29a4a86db2624a313c0766"}
```

Comment:
What commit is this?

Reply:
If I understand correctly, the `rev="3b2ef3a72df6f740bf29a4a86db2624a313c0766"` points directly to a commit.

I'm going to look into having this track a branch rather than a commit.